third_party/sl4a/gradle.properties

```diff
@@ -1,6 +1,6 @@
 # This version code implements the versioning recommendations in:
 # https://blog.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/
-VERSION_CODE=1000100
-VERSION_NAME=1.0.1-SNAPSHOT
+VERSION_CODE=1000199
+VERSION_NAME=1.0.1
```

Comment:
The version was 1.0.1-SNAPSHOT (which is maven/gradle speak for prerelease) and now 1.0.1 is released:
http://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it